res/openvpn-gui-res-cs.rc

```diff
@@ -38,8 +38,7 @@ END
 
 /* Auth Username/Password Dialog */
 ID_DLG_AUTH DIALOG 6, 18, 160, 95
-STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | DS_CENTER | DS_SETFOREGROUND
-CAPTION "OpenVPN - Ověření uživatele"
+STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER | DS_SETFOREGROUND
```

Comment:
Adding SYS_MENU style is needed to get icon displayed, but that also adds a close button. Either disable that button or add StopOpenVPN() to WM_CLOSE (the same way as how ID_CANCEL is handled). Otherwise close will lead to the management interface stuck at "waiting for hold-release".